src/navigation/launcher-navigation-directive.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.navigation.directive:pfLauncherNavigation
+ *
+ * @description
+ * Directive for rendering application launcher dropdown.
+ *
+ * @param {boolean=} isOpen Override and determine open and closed state, default: false
+ * @param {boolean=} isDisabled Disable the application launcher button, default: false
+ * @param {boolean=} isList Display items as a list instead of a grid, default: false
+ * @param {boolean=} hiddenIcons Flag to not show icons on the launcher, default: false
+ * @param {array} items List of navigation items
+ * <ul style='list-style-type: none'>
+ * <li>.title        - (string) Name of item to be displayed on the menu
+ * <li>.iconClass    - (string) Classes for icon to be shown on the menu (ex. "fa fa-dashboard")
+ * <li>.href         - (string) href link to navigate to on click
+ * <li>.tooltip      - (string) Tooltip to display for the badge
+ * </ul>
+ * @example
+ <example module="patternfly.navigation">
+   <file name="index.html">
+   <div ng-controller="launcherNavController">
+     <div pf-launcher-navigation="" items="navigationItems" is-open="isOpen" is-disabled="isDisabled" is-list="isList" hidden-icons="hiddenIcons"></div>
+   </div>
+   </file>
+   <file name="script.js">
+   angular.module('patternfly.navigation').controller('launcherNavController', ['$scope',
+     function ($scope) {
+
+       $scope.navigationItems = [
+          {
+            title: "Recteque",
+            href: "#/ipsum/intellegam/recteque",
+            tooltip: "Total number of error items",
+            iconClass: "",
+          },
+          {
+            title: "Suavitate",
+            href: "#/ipsum/intellegam/suavitate",
+            tooltip: "Total number of items",
+            iconClass: "",
+          }
+       ];
+
+       $scope.isDisabled = false;
+       $scope.isOpen = false;
+       $scope.isList = false;
+       $scope.hiddenIcons = false;
+     }]);
+   </file>
+ </example>
+ */
+angular.module('patternfly.navigation').directive('pfLauncherNavigation', [
+  function () {
+
+    'use strict';
+
+    return {
+
+      restrict: 'A',
+
+      replace: true,
+
+      scope: {
+
+        items: '&',
+        isOpen: '=?',
+        isDisabled: '&?',
+        isList: '&?',
+        hiddenIcons: '&?'
```

Comment:
one-way binds. we could change them, but it's not necessary to send the info back out. unless we're planning on using it for something?